extensions/interactions/TextInput/TextInput.html

```diff
@@ -8,7 +8,7 @@
       <schema-based-editor schema="schema" local-value="answer" label-for-focus-target="labelForFocusTarget">
       </schema-based-editor>
 
-      <button ng-click="submitAnswer(answer)" class="btn btn-default" style="margin-top: 15px;">
+      <button ng-click="submitAnswer(answer)" class="btn btn-default" style="margin-top: 0px;">
```

Comment:
Do not include units if the value is 0. E.g. margin-left: 0 instead of margin-left: 0px.